Registry/Registry.EM_COMMON

```diff
@@ -2601,6 +2573,7 @@ rconfig   integer scalar_pblmix           namelist,physics      max_domains  0
 rconfig   integer tracer_pblmix           namelist,physics      max_domains  1         h    "mix 4d tracer variables with pbl scheme 0=no 1=yes" ""
 rconfig   logical use_aero_icbc           namelist,physics     	1              .false. rh    "use_aero_icbc"  "Use GOCART climo 3D aerosols IC/BC data in Thompson-MP-Aero"  "logical flag"
 rconfig   logical use_rap_aero_icbc       namelist,physics      1              .false. r     "use_rap_aero_icbc"  "Use GOCART climo 3D aerosols IC/BC data in Thompson-MP-Aero from RAP"  "logical flag"
+rconfig   integer aer_init_opt            namelist,physics      1              0       irh   "aer_init_opt"   "surrogate to handle aerosol IC/BC data in Thompson-MP-Aero: 0=no IC/BC aerosol, 1=climo, 2=first guess"
```

Comment:
@twjuliano 
Tim,
Does "surrogate" mean that a user never specifies this option `aer_init_opt`, that this value is defined entirely inside the code?

If so, change the string `namelist,physics` to `derived`.

Reply:
Yes, you're exactly right. This is now fixed in the latest commit